roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1927,7 +1927,7 @@ def get_defaults(self, roles, deployment_type, deployment_subtype):
                                       console_use_ssl=True,
                                       console_path='/console',
                                       console_port='8443', etcd_use_ssl=True,
-                                      etcd_hosts='', etcd_port='4001',
+                                      etcd_hosts='', etcd_port='2379',
```

Comment:
is this change intentional?

Reply:
Yes, it is. I think it's an outdated default and would rather just use it than continue to override it.

Reply:
no, everywhere overrides it as far as i know.